.github/ISSUE_TEMPLATE/engineering-manager-onboarding.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Engineering Manager Onboarding
+about: A list of things to do as a new manager to become familiar with the nature of our engineering team and the context it exists in.
+---
+
+## Your second week
+
+You’ve done general people-ops and engineering team onboarding, and have casually met some team-members. Now it’s time to start onboarding as a new engineering manager.
+
+- [ ] If you’re reading this, you’re the first external engineering manager we’ve hired and your name is Adam. Yay! (If not, this will be _real_ awkward.) At Artsy we are trying to give new hires ownership over improving the onboarding documentation that they are faced with, in an effort to help the next new hire onboard more smoothly. As you are the first one, there is plenty of work to do in this area; in fact, this very document is nothing but a first draft, very much a work in progress. It will be your task to make this document into a substantially larger one, that draws from all the things you had to learn. You’re not on your own, though, your fellow peers will all be available to support you during this task. You got this!
+- [ ] The people-ops team should have signed you up for the various on-site manager trainings.
+- [ ] Your manager should have invited you to meetings relevant to managers.
+- [ ] Your manager should have scheduled 1:1s for you with the other engineering managers, including the CTO, and other leadership people.
+
+### Extended sprint rotation and meet & greet
+
+Unlike individual contributors, who rotate for a few sprints, engineering managers are expected to rotate across all teams. While your main responsibility will be to be a people manager, we expect anybody on the engineering team to understand roughly how we do things and what systems are in use at Artsy, but _mostly_ this will be a chance to work with all the engineering team members and product people such as PMs and designers. Currently there are 5 teams, so with sprints having a duration of 2 weeks that’s easily going to span your next 2 months.
+
+During this time you will be expected to perform work as any other individual contributor, however, time will be reserved for the main objectives which are:
+
+- [ ] Have 1:1s with each and every engineer (that includes tech-leads), PM, designer, and some business owners. The tech-leads and PMs should have been made aware of this objective and provide you with time to do so. You are responsible for organizing these.
+- [ ] While working with individual contributors and during your 1:1s, try to uncover, and make note of, people’s expectations that are currently not being met by the organization or their manager.
+- [ ] By the end of sprint rotation you should have a list of individual contributors that you think you should manage and have argumentation for why.
```

Comment:
@dblock Remind me, what was the motivation for this? I assumed it was meant as an exercise for him to think about how he can add value to a report as their manager, is that the case? If so, I think we can maybe drop this one and improve [the above](https://github.com/artsy/README/pull/95/files#diff-f85005588f9a5181ef9ec4d27af69920R22) to “…, try to develop an understanding of where and how you want to add value as a manager”

Reply:
While I don’t believe the intent was to use this list as if set in stone, but just for the exercise, it may give off the wrong signal.